docs/index.md

```diff
@@ -1,27 +1,20 @@
 # HIP documentation
 
 The Heterogeneous-computing Interface for Portability (HIP) API is a C++ runtime
-API and kernel language that lets developers create portable applications for AMD
-and NVIDIA GPUs from single source code.
+API and kernel language that lets developers create portable applications running
+in heterogeneous systems, using CPUs and AMD GPUs and NVIDIA GPUs from a single source code.
+HIP provides a simple marshalling language to access either the AMD ROCM back-end,
+or NVIDIA CUDA back-end, to build and run application kernels. For more information,
+see [Introduction to HIP](./understand/introduction_to_hip).
 
-For HIP supported AMD GPUs on multiple operating systems, see:
-
-* [Linux system requirements](https://rocm.docs.amd.com/projects/install-on-linux/en/latest/reference/system-requirements.html#supported-gpus)
-* [Microsoft Windows system requirements](https://rocm.docs.amd.com/projects/install-on-windows/en/latest/reference/system-requirements.html#windows-supported-gpus)
-
-The CUDA enabled NVIDIA GPUs are supported by HIP. For more information, see [GPU Compute Capability](https://developer.nvidia.com/cuda-gpus).
-
-On the AMD ROCm platform, HIP provides header files and runtime library built on top of HIP-Clang compiler in the repository [Common Language Runtime (CLR)](./understand/amd_clr), which contains source codes for AMD's compute languages runtimes as follows,
-
-On non-AMD platforms, like NVIDIA, HIP provides header files required to support non-AMD specific back-end implementation in the repository ['hipother'](https://github.com/ROCm/hipother), which translates from the HIP runtime APIs to CUDA runtime APIs.
-
-## Overview
+The HIP documentation is organized as follows:
 
 ::::{grid} 1 1 2 2
 :gutter: 3
 
 :::{grid-item-card} Install
 
+* [Linux system requirements](https://rocm.docs.amd.com/projects/install-on-linux/en/latest/reference/system-requirements.html#supported-gpus)
```

Comment:
@randyh62 The original link point us to the supported GPUs and you also remove the Windows supported GPUs.  Also removed the CUDA enabled GPUs list: "The CUDA enabled NVIDIA GPUs are supported by HIP. For more information, see [GPU Compute Capability](https://developer.nvidia.com/cuda-gpus)."

We can add requirements links to install, but we need to also have a link about the supported GPUs. We can add that to Install or Reference. Maybe link the requirements at install and add the supported GPUs to reference?